tests/functional/aws-node-sdk/test/bucket/get.js

```diff
@@ -134,6 +134,24 @@ describe('GET Bucket - AWS.S3.listObjects', () => {
                 .catch(done);
         });
 
+        it('should return multiple common prefixes', done => {
+            const s3 = bucketUtil.s3;
+            const Bucket = bucketName;
+            const objects = [
+                { Bucket, Key: 'testB%' },
+                { Bucket, Key: 'testB%test.json', Body: '{}' },
+                { Bucket, Key: 'testA%' },
+                { Bucket, Key: 'testA%test.json', Body: '{}' },
+                { Bucket, Key: 'testA%test%test.json', Body: '{}' },
+            ];
+
+            Promise
+                .mapSeries(objects, param => s3.putObjectAsync(param))
+                .then(() => s3.listObjectsAsync({ Bucket, Delimiter: '%' }))
+                .then(() => done())
```

Comment:
Shouldn't you be checking that the result matches what you expect?  Also, please return the promise and you don't need to use "done" with promises.